src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -1557,4 +1730,65 @@ string TriangleCountExecutor::countCentralStoreTriangles(std::string aggregatorH
 int TriangleCountExecutor::getUid() {
     static std::atomic<std::uint32_t> uid { 0 };
     return ++uid;
+}
+
+int TriangleCountExecutor::collectPerformaceData(PerformanceSQLiteDBInterface perDB, std::string graphId, std::string command, std::string category,
+                                                 int partitionCount, std::string masterIP) {
+
+    int elapsedTime = 0;
+    time_t start;
+    time_t end;
+    PerformanceUtil performanceUtil;
+    performanceUtil.init();
+    Utils utils;
+
+    std::vector<Place> placeList = performanceUtil.getHostReporterList();
+    std::string slaCategoryId = performanceUtil.getSLACategoryId(command,category);
+
+    std::vector<Place>::iterator placeListIterator;
+
+    for (placeListIterator = placeList.begin(); placeListIterator != placeList.end(); ++placeListIterator) {
+        Place place = *placeListIterator;
+        std::string host;
+
+        std::string ip = place.ip;
+        std::string user = place.user;
+        std::string serverPort = place.serverPort;
+        std::string isMaster = place.isMaster;
+        std::string isHostReporter = place.isHostReporter;
+        std::string hostId = place.hostId;
+        std::string placeId = place.placeId;
+
+        if (ip.find("localhost") != std::string::npos || ip.compare(masterIP) == 0) {
+            host = ip;
+        } else {
+            host = user + "@" + ip;
+        }
+
+
+        if (isMaster.find("true") != std::string::npos || host == "localhost" || host.compare(masterIP) == 0) {
+
```

Comment:
We can remove this empty block with negation operator

Reply:
Change is done Dr Miyuru

Reply:
Ok